iroh/src/client/authors.rs

```diff
@@ -23,11 +23,40 @@ where
     C: ServiceConnection<RpcService>,
 {
     /// Create a new document author.
+    ///
+    /// You likely want to save the returned [`AuthorId`] somewhere so that you can use this author
+    /// again.
+    ///
+    /// If you need only a single author, use [`Self::default`].
     pub async fn create(&self) -> Result<AuthorId> {
         let res = self.rpc.rpc(AuthorCreateRequest).await??;
         Ok(res.author_id)
     }
 
+    /// Returns the default document author of this node.
+    ///
+    /// On persistent nodes, the author is created on first start and its public key is saved
+    /// in the data directory.
+    ///
+    /// The default author can be set with [`Self::set_default`].
+    pub async fn default(&self) -> Result<AuthorId> {
+        let res = self.rpc.rpc(AuthorGetDefaultRequest).await?;
+        Ok(res.author_id)
+    }
+
+    /// Set the node-wide default author.
+    ///
+    /// If the author does not exist, an error is returned.
+    ///
+    /// This is a noop on memory nodes. On peristent node, the author id will be saved to a file in
```

Comment:
why is this a noop, shouldn't this change the default on a memory node as well?

Reply:
Fixed the comment.